src/components/inspector/form-multiselect.vue

```diff
@@ -61,6 +71,27 @@ export default {
       }
       this.$emit('input', value);
     },
+    value: {
+      immediate: true,
+      handler() {
+        this.selected = null;
+        // Load selected item.
+        if (this.value) {
+          this.options.forEach(item => {
+            if (item && item.value && item.value === this.value) {
+              this.selected = item;
```

Comment:
If you refactor the `multiselect` component to use `custom-label`, like in the `page-select.vue` file, the two watchers can be removed. Like:

```
<multiselect
  v-bind="$attrs"
  v-on="$listeners"
  ...
  :options="options.map(option => option.value)"
  :custom-label="getLabelFromValue"
>
...
  ...
  inheritAttrs: false,
  ...
  methods: {
    getLabelFromValue(value) {
      const selectedOption = this.options.find(option => option.value == value);
      return selectedOption ? selectedOption.content : null;
    },
  },
  ...
```

Reply:
done